@xen-orchestra/lite/src/stories/web-core/input/ui-textarea.story.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties, settings }"
+    :params="[
+      model().type('string').preset(''),
+      prop('accent').enum('brand', 'warning', 'danger').required().preset('brand').widget(),
```

Comment:
Need to rebase and change `brand` to `info`
```suggestion
      prop('accent').enum('info', 'warning', 'danger').required().preset('info').widget(),
```